test/cases/fixes/incompleteTypes/implicitGenerics/incompleteImplicitClassGenerics/expected.ts

```diff
@@ -21,7 +21,7 @@
 
 
 
-    class ExtendsBaseWithOneInterface extends BaseWithOneGeneric<OneInterface> {
+    class ExtendsBaseWithOneInterface extends BaseWithOneGeneric<{ property: string }> {
```

Comment:
No, the object literal printing should have a lower precedence than direct known interface names...